tests/common/utilities.py

```diff
@@ -5,6 +5,7 @@
 import inspect
 import ipaddress
 import logging
+import pytest
```

Comment:
`pytest` is a third-party import. Per [PEP8](https://pep8.org/#imports), it should be included in the second group. Note that `import re` is not in alphabetical order and `from io import BytesIO` is also in the wrong group. It _should_ read:

```python
import collections
import inspect
import ipaddress
import logging
import re
import six
import sys
import threading
import time
from io import BytesIO

import pytest
from ansible.parsing.dataloader import DataLoader
from ansible.inventory.manager import InventoryManager
from ansible.vars.manager import VariableManager

from tests.common.cache import cached
from tests.common.cache import FactsCache
```